### ISSUE TITLE
No content in the help and feedback page

### ISSUE THREAD

**Issue by biswajit150803** [akshatmittal61/planner#22]
When the help and report a bug option is clicked from hamburger menu the page shows oops error and also send us a feedback option is not working.So i want to work on that

**Comment by akshatmittal61** [comment]
Yes sure, you can build those pages and I will review and merge your work after you've raised a PR for the same.

**Comment by sayak22** [comment]
I'm interested in resolving this issue, please assign it to me.

**Comment by akshatmittal61** [comment]
Yes sure, you may work on this and raise a PR for the same. Your work will be merged after review and approval.
